evalml/tests/pipeline_tests/test_time_series_pipeline.py

```diff
@@ -185,84 +258,101 @@ def test_fit_drop_nans_before_estimator(
         (TimeSeriesMulticlassClassificationPipeline, "Random Forest Classifier"),
     ],
 )
-@patch("evalml.pipelines.components.RandomForestClassifier.fit")
 @patch("evalml.pipelines.components.RandomForestClassifier.predict")
-@patch("evalml.pipelines.components.RandomForestRegressor.fit")
 @patch("evalml.pipelines.components.RandomForestRegressor.predict")
-@patch(
-    "evalml.pipelines.TimeSeriesClassificationPipeline._decode_targets",
-    side_effect=lambda y: y,
-)
-def test_predict_pad_nans(
-    mock_decode_targets,
+def test_predict_and_predict_in_sample(
```

Comment:
This test and test_compute_estimator_features capture are the most important modifications to this file. They verify that the features passed to the estimator respect the window described in the user guide. Most of the other modifications on this file are just making changes so that they wouldn't break given the new api but the behavior is the same.